ITensorGaussianMPS/src/ITensorGaussianMPS.jl

```diff
@@ -8,10 +8,15 @@ using LinearAlgebra
 import LinearAlgebra: Givens
 
 export slater_determinant_to_mps,
+  correlation_matrix_to_mps,
   slater_determinant_to_gmps,
+  correlation_matrix_to_gmps,
   hopping_hamiltonian,
+  pairing_hamiltonian,
   slater_determinant_matrix,
-  slater_determinant_to_gmera
+  slater_determinant_to_gmera,
+  Pairing,
+  ConservingNf
```

Comment:
I think we shouldn't export these since they should be mostly hidden from external users.

Reply:
Ok.